src/renderingManager.js

```diff
@@ -30,15 +31,16 @@ export function newRenderingManager(win, environment) {
    */
   let renderAd = function(doc, dataObject) {
     const targetingData = utils.transformAuctionTargetingData(dataObject);
-
-    if (environment.isMobileApp(targetingData.env)) {
-      renderAmpOrMobileAd(targetingData.cacheHost, targetingData.cachePath, targetingData.uuid, targetingData.size, targetingData.hbPb, true);
-    } else if (environment.isAmp(targetingData.uuid)) {
-      renderAmpOrMobileAd(targetingData.cacheHost, targetingData.cachePath, targetingData.uuid, targetingData.size, targetingData.hbPb);
+    const freestar = new Freestar(targetingData);
+    if (environment.isMobileApp(freestar.env)) {
+      freestar.appBidTrack(freestar.uuid);
```

Comment:
Is this potentially fired too early ?  Should the call be nested a level lower where it fires other trackers  well ?